src/styles/star.css

```diff
@@ -0,0 +1,22 @@
+.stars__star {
+  display: flex;
+}
+
+.stars__star-review {
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  
+  background-image: url(../images/star.svg);
+  border-radius: 0.5px;
+}
+
+.stars__star-review:last-child {
+  margin-right: 0;
+}
+
+.stars__star :nth-child(-n + 4) {
```

Comment:
use all class selectors from Stars task, so that changing the class will change the amount of stars